ztp/ztp-policy-generator/kustomize/plugin/policyGenerator/v1/policygenerator/siteConfig/siteConfigBuilder.go

```diff
@@ -53,6 +53,13 @@ func (scbuilder *SiteConfigBuilder) Build(siteConfigTemp SiteConfig) (map[string
 		if cluster.ClusterName == "" {
 			return clustersCRs, errors.New("Error: Missing cluster name at site " + siteConfigTemp.Metadata.Name)
 		}
+		if cluster.NetworkType == "" {
```

Comment:
Having the CRD definition as Enum with default value will avoid those checks.

Reply:
Removed checks since those are now at CRD level.